src/EventStream/Event.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Casper\EventStream;
+
+class Event
+{
+    private ?string $id = null;
+
+    private ?string $name = null;
+
+    private ?array $data = null;
```

Comment:
@RomanovSci we need to add typing here. I guess, the most convenient way would be to have event-specific event classes. The parsing could be moved to the `EventStream`.

Note, that events include the standard Casper types inside of them.